NAMESPACE

```diff
@@ -4,20 +4,20 @@ S3method("*",greta_kernel)
 S3method("+",greta_kernel)
 S3method(print,greta_kernel)
 export(bias)
+export(constant)
+export(cosine)
 export(expo)
 export(gp)
 export(linear)
 export(mat12)
 export(mat32)
 export(mat52)
 export(periodic)
+export(polynomial)
 export(project)
+export(rational_quadratic)
 export(rbf)
-export(white)
```

Comment:
did you mean to de-export the white kernel?

Reply:
I did because I didn't understand why it was returning all zeros in some cases. All clear now, added back in.